distribution/docker/src/docker/bin/curl

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
```

Comment:
The script won't execute without a shebang, and I don't see how it can throw off anyone else's scripting unless they're doing something daft like `file /usr/bin/curl` (and `file` isn't present in the final image anyway).

Reply:
If we know `wget` has funny behavior do we still want to go this route of pointing them at that tool?

Reply:
I oscillated about this, but in the end I removed it. If people expect to find `curl`, I'd rather it be missing and they complained. Then maybe we have chance to learn more about what they're trying to do.